### ISSUE TITLE
Ensure File References are Resolvable

### ISSUE THREAD

**Issue by brianrufgsa** [usnistgov/OSCAL#181]
# User Story:

As an OSCAL content creator, I can use relative and absolute references in OSCAL content to other resources (e.g., schema, other OSCAL content, etc.) using `href` attributes. OSCAL content users need these references to be reliable in production moving forward. This includes references to external OSCAL style sheets, profiles, catalogs, and other files.

## Goals:

- Ensure that all file references are resolvable in OSCAL content in the `content` directory. Relative locations should be used where possible. (@brianrufgsa)
- Remove any unneeded references. (@brianrufgsa)

## Dependencies:

- Issue #192 should be done first to ensure that all locations are stable before references are updated.

## Acceptance Criteria

- [ ] All references in content have been either removed if possible, or fixed and made resolvable.

**Comment by david-waltermire-nist** [comment]
@brianrufgsa: @wendellpiez and @david-waltermire-nist will meet on 6/1 to work out locations for schema, schematron, etc. We can provide a punchlist of issues to resolve for this ticket.

**Comment by brianrufgsa** [comment]
Pull request #207 addresses this. 
Establishes htts://githib.com/usnistgov/OSCAL/pub for all published files.
Further establishes ./nist.gov and ./fedramp.gov for 800-53 and FedRAMP files respectively.

#Example:
The following would appear in the 800-53 High and FedRAMP High Profiles
`<import href="htts://githib.com/usnistgov/OSCAL/pub/nist.gov/NIST_SP-800-53_rev4_catalog.xml"> `

**Comment by brianrufgsa** [comment]
**June 21 Status**: The modification is made and the pull request is pending. 
The official location for all _published_ OSCAL files (Profiles, catalogs, and supporting files) is now:
`https://githib.com/usnistgov/OSCAL/pub`

I created a `nist.gov` sub-folder for the NIST SP 800-53 r4 catalog and related profiles.
I also created a `fedramp.gov` sub-folder for the FedRAMP High, Moderate, and Low profiles.

**Comment by david-waltermire-nist** [comment]
# 6/21/2018 Status
- Once PR #201 is accepted, a new PR needs to be submitted with these changes. This will ensure all schema references are consistent with the new directory structure.

**Comment by iMichaela** [comment]
## June 28 Status Meeting Update ##
Addressed in the PR #201. Will be completed when this PR is addressed.

**Comment by iMichaela** [comment]
## July 18 Status Update (Sprint 12 Acceptance Meeting) ##
PR #201 is awaiting review from @david-waltermire-nist and/or @iMichaela

**Comment by iMichaela** [comment]
## 7/25/2018 Status Meeting ##
PR #201 was accepted, @wendellpiez needs to submit a **new PR** with these changes. This will ensure all schema references are consistent with the new directory structure.

**Comment by david-waltermire-nist** [comment]
The ```examples``` directory has been moved to the ```content``` directory. This will be verified as part of #185.

**Comment by david-waltermire-nist** [comment]
This has been addressed in PR #201. Closing this and leaving #185 open related to PR #225.
